doc/man3/RAND_get0_primary.pod

```diff
@@ -0,0 +1,78 @@
+=pod
+
+=head1 NAME
+
+RAND_get0_primary,
+RAND_get0_public,
+RAND_get0_private
```

Comment:
Should OSSL_ or OPENSSL_ prefix be used here for these new API calls? Or even OPENSSL_get0_primary_rand or OPENSSL_CTX_get0_primary_rand?

Reply:
These are new API calls, but they are added to the existing `RAND` API. That's why no additional prefix was added.

Reply:
I agree with @paulidale that these calls do not belong to the EVP_RAND API. On the other hand I'd prefer keeping the RAND_ API basically sealed i.e. not add anything to it as the RAND prefix is quite generic. For that reason I'd prefer these calls to be OPENSSL_get0_primary_rand or OPENSSL_CTX_get0_primary_rand. But I do not insist on that.

Reply:
I'm okay to change the names.  If this gets merged for the alpha, it will need to be a separate PR but that's not a problem.

Reply:
We need to agree on the naming first.  OPENSSL_get0_primary_rand() is okay.  Would OPENSSL_RAND_get0_primary() be more appropriate?